static/scss/detail/header.scss

```diff
@@ -77,101 +77,6 @@
     }
   }
 
-  .action-button {
-    font-size: x-large;
-    font-weight: 400;
-  }
-
-  .enroll-button {
-    display: inline-flex;
-    margin: 0;
-    border-radius: 5px;
-    background-color: $primary;
-    color: white;
-    padding: 20px 40px;
-    text-transform: uppercase;
-    cursor: pointer;
-    font-weight: 700;
-
-    &.enrolled {
-      border: 4px solid $primary;
-      background-color: white;
-      color: $primary;
-    }
-
-    &:hover {
-      // underline is at different levels since we centered the text and checkmark vertically, so it looks strange
-      text-decoration: none;
-    }
-  }
-
-  .enroll-dropdown {
-    .dropdown-toggle::after {
-      display: none;
-    }
-
-    .login-popup {
-      max-width: 350px;
-      margin-left: 20px;
-      margin-right: 20px;
-      padding: 20px;
-      font-weight: 600;
-      font-size: 20px;
-
-      .close-btn {
-        position: absolute;
-        top: 0;
-        right: 0;
-        padding: 5px;
-        margin: 20px 7px 0 0;
-        font-size: 72px;
-        font-weight: 300;
-        background-color: #fff !important;
-        color: #000;
-        min-width: unset;
-        height: unset;
-        cursor: pointer;
-      }
-
-      h4 {
-        font-weight: 700;
-        font-size: 22px;
-      }
-
-      .popup-buttons {
-        display: flex;
-        flex-direction: row;
-        justify-content: space-between;
-        margin-top: 20px;
-
-        .link-button {
-          border: 2px solid $light-blue;
-          border-radius: 10px;
-          font-weight: 600;
-          font-size: 24px;
-
-          &.sign-in-link {
-            color: white;
-            background-color: $light-blue;
-          }
-
-          &.create-account-link {
-            color: black;
-          }
-        }
-      }
-    }
-
-    .triangle {
-      width: 15px;
-      height: 15px;
-      position: absolute;
-      top: -7px;
-      background: white;
-      transform: rotate(45deg);
-    }
-  }
-
```

Comment:
I think you'll need to revert this as well, for the enroll now button in the hero section.

Reply:
From the story description, I assumed, We need to stick the `Enroll Now` button to sab-navigation bar. 
So do you mean, we need to keep `Enroll Now / Apply Now` button state in both `Hero` + `Subnavigation` section ?

Reply:
That’s right. We want the button in both places.